### ISSUE TITLE
cannot seem to get the import to work correctly 

### ISSUE THREAD

**Issue by chadmott** [justinlettau/azure-ad-verify-token#45]
## Type

What kind of issue is this?

```
[x ] Bug report.
[ ] Feature request.
```

## Description the Issue

No matter how I try this script (vanilla node.js or typescript) I cannot run verify b/c of module issues.

When running as ts-node, I get:

```
require() of ES Module ...node_modules/azure-ad-verify-token/dist/index.js from index.ts not supported.
to a dynamic import() which is available in all CommonJS modules.
```
when running in javascript 
```
Cannot find module azure-ad-verify-token/dist/config' imported from ../node_modules/azure-ad-verify-token/dist/index.js
```

Todo

Try to use the library as illustrated 

n/a

## Other Information

n/a

**Comment by justinlettau** [comment]
This package is an ESM now. This info [here](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) should be what you need. Hope that helps!

**Comment by chadmott** [comment]
Ya I did that.. and it still doesn’t work. Do you have a working gist I could try to see what I’ve done wrong?

**Comment by chadmott-deloitte** [comment]
specifically, when running in node 18.6 with typescript 4.7.4...

tsconfig.json

```
{
    "compilerOptions": {
      "sourceMap": true,
      "outDir": "dist",
      "strict": true,
      "lib": ["esnext"],
      "esModuleInterop": true,
      "resolveJsonModule": true,
      "suppressImplicitAnyIndexErrors": true,
      "noImplicitAny": false,
      "module": "NodeNext",
    }
  }
  
```
package.json

```
  "type": "module",
  "exports": "./index.ts"
```

in a project with many other dependencies, yours is the only one I get TSC errors on...

```
node_modules/azure-ad-verify-token/dist/index.d.ts:1:38 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './config.js'?

1 export { getConfig, setConfig } from './config';
                                       ~~~~~~~~~~

node_modules/azure-ad-verify-token/dist/index.d.ts:2:24 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './verify.js'?

2 export { verify } from './verify';
                         ~~~~~~~~~~

node_modules/azure-ad-verify-token/dist/index.d.ts:3:39 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './interfaces.js'?

3 export { Config, VerifyOptions } from './interfaces';
                                        ~~~~~~~~~~~~~~


Found 3 errors in the same file, starting at: node_modules/azure-ad-verify-token/dist/index.d.ts:1

```

is this meant to run in node?

**Comment by chadmott** [comment]
specifically, when running in node 18.6 with typescript 4.7.4...

tsconfig.json
```
{
    "compilerOptions": {
      "sourceMap": true,
      "outDir": "dist",
      "strict": true,
      "lib": ["esnext"],
      "esModuleInterop": true,
      "resolveJsonModule": true,
      "suppressImplicitAnyIndexErrors": true,
      "noImplicitAny": false,
      "module": "NodeNext",
    }
  }
  ```
package.json
```
  "type": "module",
  "exports": "./index.ts"

```
in a project with many other dependencies, yours is the only one I get TSC errors on...
```

~/tsc

Found 3 errors in the same file, starting at: node_modules/azure-ad-verify-token/dist/index.d.ts:1

node_modules/azure-ad-verify-token/dist/index.d.ts:1:38 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './config.js'?

1 export { getConfig, setConfig } from './config';
                                       ~~~~~~~~~~

node_modules/azure-ad-verify-token/dist/index.d.ts:2:24 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './verify.js'?

2 export { verify } from './verify';
                         ~~~~~~~~~~

node_modules/azure-ad-verify-token/dist/index.d.ts:3:39 - error TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './interfaces.js'?

3 export { Config, VerifyOptions } from './interfaces';
                                        ~~~~~~~~~~~~~~

```


is this meant to run in node? typescript will not compile this. If you could link to a repo/project that uses typescript and node that would likely be very helpful

**Comment by justinlettau** [comment]
@chadmott Thank you for the extra details! I believe I found the issue .. and it should be fixed in v2.0.1 🎉  Do you mind trying and letting me know? Thanks!

**Comment by chadmott** [comment]
Thanks @justinlettau 2.0.1 resolved my issue! appreciate the quick response and helpful links.
